lib/castle-rb/client.rb

```diff
@@ -11,17 +11,30 @@ def initialize(request, response)
     end
 
     def identify(args)
-      @api.request('identify', args)
+      @api.request('identify', args) unless do_not_track?
     end
 
     def authenticate(args)
       @api.request('authenticate', args)
```

Comment:
the problem with implementing it there is that you will want to work with the response (not fire and forget). So the code should look like this:
```
if castle.do_not_track?
  sign_in(user)
else
  verdict = castle.authenticate("login.succeeded")
  ...
```
happy to discuss a better solution! @wallin